test/e2e/karmadactl_test.go

```diff
@@ -1154,6 +1154,163 @@ var _ = ginkgo.Describe("Karmadactl options testing", func() {
 	})
 })
 
+var _ = framework.SerialDescribe("Karmadactl taint testing", ginkgo.Labels{NeedCreateCluster}, func() {
+	ginkgo.Context("Test karmadactl taint command with different effects", func() {
+		var (
+			newClusterName, clusterContext        string
+			homeDir, kubeConfigPath, controlPlane string
+			taintKey                              = "environment"
+			taintProductionValue                  = "production"
+			taintTestValue                        = "test"
+		)
+
+		ginkgo.BeforeEach(func() {
+			newClusterName = "member-e2e-" + rand.String(3)
+			homeDir = os.Getenv("HOME")
+			kubeConfigPath = fmt.Sprintf("%s/.kube/%s.config", homeDir, newClusterName)
+			controlPlane = fmt.Sprintf("%s-control-plane", newClusterName)
+			clusterContext = fmt.Sprintf("kind-%s", newClusterName)
+		})
+
+		ginkgo.BeforeEach(func() {
+			ginkgo.By(fmt.Sprintf("Creating cluster: %s", newClusterName), func() {
```

Comment:
Can we test with an existing cluster? Creating a cluster takes a certain period of time, which prolongs the overall test execution time.

Reply:
https://github.com/karmada-io/karmada/pull/5244#issuecomment-2268029145

I am sorry, I have seen this comment, it makes sense.